### ISSUE TITLE
Monorepo

### ISSUE THREAD

**Issue by ratson** [Daplie/node-letsencrypt#60]
There are many related packages, it is quite hard to follow the source code.
Is there any interest to convert them into monorepo?
The `babel` project has listed out [the pros and cons](https://github.com/babel/babel/blob/master/doc/design/monorepo.md).

I could help if you are interested in doing so.

**Comment by coolaj86** [comment]
I would love help in the area of following the source code - meaning comments, documentation, etc.

I completely agree with your reasons. We're a small team with a boatload of work to do and it's hard to keep everything up to date.

However, I like the SEO benefits of having multiple repos.

If you wanted to help in creating a gh-pages site that highlights the individual modules then I would definitely be open to moving everything into one repo.
